<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../.designer/backup/setting_button.png</normaloff>../../../.designer/backup/setting_button.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>421</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QStackedWidget
{
	border-radius: 9px;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_3"/>
    <widget class="QWidget" name="page_2">
     <widget class="QListWidget" name="sources">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>391</width>
        <height>370</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Yahei</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget 
{
    background-color: #e7e7e7;
    border: 1px solid #eaeaea;
    font: 11pt &quot;Microsoft Yahei&quot;;
	outline: 0px;
	padding-top: 1px;
}
QListWidget::item 
{
    background-color: transparent;
    color: #000000;
    border: 0px;
    padding-left: 5px;
    height: 25px;
}
QListWidget::item:selected 
{
    color: #000000;
    background-color: #d7d7d7;
}</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>391</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	background-color: #d7d7d7;
}</string>
      </property>
      <widget class="QPushButton" name="add_source">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: #eaeaea;
	border: 1px solid #bcbcbc;
}</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="delete_source">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>0</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: #eaeaea;
	border: 1px solid #bcbcbc;
}</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>设置Gallery音乐的文件路径:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancel_button">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>460</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	background-color: #d7d7d7;
	outline: none;
	border: 1px solid #eaeaea;
	border-radius: 4px;
}</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_button">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>460</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	background-color: #d7d7d7;
	outline: none;
	border: 1px solid #eaeaea;
	border-radius: 4px;
}</string>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="avatar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="username">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>55</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="save">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>55</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>150</width>
      <height>500</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Yahei</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget 
{
    background-color: #e7e7e7;
    border: none;
	border-raduis: 10px;
    font: 10.5pt &quot;Microsoft Yahei&quot;;
	outline: 0px;
	padding-top: 50px;
}
QListWidget::item 
{
    background-color: transparent;
    color: #000000;
    border: 0px;
    padding-left: 5px;
    height: 38px;
}
QListWidget::item:selected 
{
    color: #ffffff;
    background-color: #808080;
	border-radius: 9px;
}</string>
    </property>
    <item>
     <property name="text">
      <string>来源管理</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>资料设置</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>10</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color: #e7e7e7;
	border-right: 1px solid #eaeaea;
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>0</y>
      <width>9</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color: #e7e7e7;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI Semibold</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>关于musicGallery</string>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
    </font>
   </property>
  </action>
  <action name="actionconfig">
   <property name="text">
    <string>配置音乐文件夹</string>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
